packages/utilities/src/helpers/formatStatus.ts

```diff
@@ -0,0 +1,5 @@
+import { capitalizeAllWords } from './capitalize';
+
+export const getFormattedStatus = (status: string): string => {
+  return capitalizeAllWords(status.replace(/_/g, ' '));
```

Comment:
I think capitalizeAllWords was recently modified to handle delimiters (e.g. `s.split(delimiter).map(capitalize).join(' ')`) and we should be able to do the following here without the `replace`:

```suggestion
  return capitalizeAllWords(status, '_'));
```

This utility is fine, because it's clear what it's doing and avoids us having to pass in the second '_' prop everywhere, so I don't mind it existing rather than update the use of capitalizeAllWords everywhere directly.

Reply:
There are some places in the codebase where status status don't have an `_`

For example:
https://github.com/linode/manager/blob/f3639f96a21dd0f18d7559a236b5308578a73156/packages/api-v4/src/images/types.ts#L11-L12

Makes me wonder if we should keep the `replace` for robustness. Or maybe just use `capitalizeAllWords` globally? What do you guys think?

Reply:
Keeping the `replace` adds a bit of extra robustness, especially for cases without underscores, so the current approach seems like a reasonable and flexible choice—what are your thoughts?

Reply:
> There are some places in the codebase where status status don't have an _

Oh, good point! Then current approach is fine, imo. When handling statuses, devs might remember a util that has status in the name more easily than capitalizeAllWords.